src/api/agreement/helpers/accept-offer.test.js

```diff
@@ -107,3 +107,14 @@ describe('acceptOffer', () => {
     await expect(acceptOffer(agreementId)).rejects.toEqual(boomError)
   })
 })
+
+describe('getFirstPaymentDate', () => {
+  test('returns correct first payment date', () => {
+    expect(getFirstPaymentDate('2024-02-15')).toBe('May 2024')
+    expect(getFirstPaymentDate('2024-02-31')).toBe('June 2024')
+  })
+
+  test('handles invalid date string gracefully', () => {
```

Comment:
Can't we ensure that the start `agreementStartDate` is always a valid date?
What would be the consequence of having an empty string vs 'Invalid Date' string as the value? Both are invalid dates correct?

Apologies for what may be a really stupid question. These stupid questions will hopefully fade away as time goes by.